medusa/tv/series.py

```diff
@@ -559,22 +559,19 @@ def last_update_indexer(self):
     @property
     def prev_aired(self):
         """Return last aired episode ordinal."""
-        today = datetime.date.today().toordinal()
-        if not self._prev_aired or self._prev_aired > today:
-            if self._last_update_indexer >= today:
-                self._prev_aired = self.prev_episode()
+        self._prev_aired = self.prev_episode()
         return self._prev_aired
 
     @property
     def next_aired(self):
         """Return next aired episode ordinal."""
         today = datetime.date.today().toordinal()
         if not self._next_aired or self._next_aired < today:
-            if self._last_update_indexer >= today:
-                self._next_aired = self.next_episode()
+            self._next_aired = self.next_episode()
         return self._next_aired
 
     @property
+    @time_cache
```

Comment:
1h seems a bit short?

Reply:
Your right. I intended to use 12 hours. Let's use 12 for both?